modules/installation-initializing.adoc

```diff
@@ -263,6 +263,15 @@ files that the installation program creates.
 When specifying the directory:
 * Verify that the directory has the `execute` permission. This permission is required to run Terraform binaries under the installation directory.
 * Use an empty directory. Some installation assets, such as bootstrap X.509 certificates, have short expiration intervals, therefore you must not reuse an installation directory. If you want to reuse individual files from another cluster installation, you can copy them into your directory. However, the file names for the installation assets might change between releases. Use caution when copying installation files from an earlier {product-title} version.
++
+[NOTE]
+=====
+Always delete `.powervs` directory to avoid reusing a stale configuration.
```

Comment:
`~/.powervs`
Just `.powervs` assumes the PWD is $HOME

Reply:
Updated.